crates/turbopack-ecmascript/js/src/runtime.js

```diff
@@ -124,6 +124,18 @@
     }
   }
 
+  /**
+   * Adds the getters to the exports object
+   *
+   * @param {Exports} exports
+   * @param {Record<string, any>} props
+   */
+  function cjs(exports, props) {
+    for (const key in props) {
+      defineProp(exports, key, { get: () => props[key], enumerable: true });
```

Comment:
Can CJS export a `default`? Should we skip if `key === "default"`?

Reply:
`foo.cjs`:
```js
module.exports = 1;
```

`bar.cjs`:
```js
const foo = require('./foo.cjs')

console.log(foo)
```

`node bar.cjs`:

```
1
```

Seems like `default` is created if someone uses ESM and transpile it to CJS, but not by default

Reply:
I think we should skip it, as people may use `export default Foo` and publish it as a CJS package.

Reply:
I think the test should be:

```js
// foo.cjs
module.exports = 'default';
// or exports.default = 'default';


// bar.mjs
export * from './foo.cjs';


// entry.mjs
import * as bar from './bar.mjs';
console.log(bar);
```

In which case, the default is not propagated from `foo.cjs` through `bar.mjs`.

Reply:
It prints

```
[Module: null prototype] {  }
```